### DISCUSSION TITLE
Netgear A6150 working on Ventura - possible to remove the WPS search window on startup?

### DISCUSSION THREAD

**Discussion by coda1144** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#316] [Q&A]
Hi!
I tried this driver for my 2012 Macbook Pro that I've used OpenCore Legacy Patcher to update to Ventura. Everything except my USB wifi adapter from Netgear works as intended and using this driver I got that to work as well. 

The small issue I have is that I can't find a way to remove the WPS Search window from appearing on startup. Is there a way to remove this?

In any case, thanks Chris1111 for the driver. I would've had to downgrade again to Catalina if it wasn't for this driver.

Cheers

**Comment by chris1111** [comment]
I don't understand why WPS wants to activate at startup? Have you tried to activate it using the router button?

**Reply by coda1144** [reply]
No I've just connected using password from the app in the status bar and that worked. But every time I restart the WPS search window pops up and I have to close it otherwise it just idles and keeps searching or wanting me to connect via WPS. The app does connect to my wireless router perfectly every time otherwise so it would be great to find a way to turn the WPS-window off.
![wps-search](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/150078154/b3da1b35-3d3a-4aeb-8d76-85b6ed33fe81)

**Comment by chris1111** [comment]
I have 9 USB adapter and I have never had this problem in any version of macOS.
so the problem comes from your macOS install and I have no idea what to do about this problem sorry.

**Reply by coda1144** [reply]
It's all good. I'll try and reinstall the driver and see if that fixes it, but first I'll try connecting to the wifi using WPS and see if it's happy after that.
